pkg/chunked/internal/compression.go

```diff
@@ -51,6 +51,10 @@ type FileMetadata struct {
 
 	// internal: computed by mergeTOCEntries.
 	Chunks []*FileMetadata `json:"-"`
+	// SkipSetAttrs is set when the file attributes must not be
+	// modified, e.g. it is a hard link from a different source,
+	// or a composefs file.
+	SkipSetAttrs bool
```

Comment:
Is this really intended to be maintained in the TOC?

If not, can this field be tracked internally inside `pkg/chunked` only?

Reply:
yeah, let's clean this up and remove it from the TOC definition.

What do you think about the new version?